### PR TITLE
discovered a unique problem with factory girl

### PR DESCRIPTION
**Pull request by brandonrandall**
in the test, don't create an attribute of something that is in the
factory already

### CODE REVIEW

File: spec/features/coach_dashboard_spec.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                          password: "123")
+    profile = CoachProfile.create(institution: "Eagles",
+                                  email: "1@gmail.com",
+                                  phone_number: "123",
+                                  coach_id: coach.id)
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(coach)
+    visit "/coach"
+    expect(current_path).to eq("/coach")
+    expect(page).to have_content("Hello #{coach.type} #{coach.first_name}")
+    expect(page).to have_css(".institution")
```
Comment (mdevoe12) [inline spec/features/coach_dashboard_spec.rb:42]: @brandonrandall 

The test on Line 41 looks solid. Can you test for have_content for institution, email and phone. Thanks!

**Comment by thedanielvogelsang** [comment]
Is this a merge-able branch @brandonrandall ?

File: app/views/coach_dashboard/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<h1>Hello <%= current_user.type %> <%= current_user.first_name %></h1>
+
+<p class="institution">
+  <h2>Institution: <%= current_user.profile.institution %></h1>
```
Comment (thedanielvogelsang) [inline app/views/coach_dashboard/show.html.erb:4]: checkout your h2 tag... its closed with /h1. Also, no need to wrap h2's in <p> tags. What about a <div>?

File: app/views/coach_dashboard/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+</p>
+
+<p class="email">
```
Comment (thedanielvogelsang) [inline app/views/coach_dashboard/show.html.erb:7]: same

File: app/views/coach_dashboard/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  <h2>Email: <%= current_user.email%></h1>
+</p>
+
+<p class="phone_number">
+  <h2>Phone Number: <%= current_user.profile.phone_number %></h1>
+</p>
+  <%# binding.pry %>
```
Comment (thedanielvogelsang) [inline app/views/coach_dashboard/show.html.erb:14]: binding.pry still in the code

File: spec/features/coach_dashboard_spec.rb [hunk 5]
```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+# describe "user searches for stations by zip" do
```
Comment (thedanielvogelsang) [inline spec/features/coach_dashboard_spec.rb:3]: What's all this commented-out code @brandonrandall ?

**Review by thedanielvogelsang — CHANGES_REQUESTED** [review]
There are several changes I think that should be made before we consider merging this branch. See pending review changes for my comments.

**Pull request closed without merging**